src/schema/imported/manifest.json

```diff
@@ -355,6 +355,37 @@
         }
       }
     },
+    "WebExtensionDictionaryManifest": {
```

Comment:
This doesn't have any special handling yet, I opened https://github.com/mozilla/addons-linter/issues/1984 to track that work.